### ISSUE TITLE
Should we include `TODO` documents?

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/easystats#330]
Inspired by @rempsyc's list: https://rempsyc.remi-theriault.com/TODOS.html

Writing such a document would be a good way to force ourselves to think as to which of the pending features are more important, and it might also provide users with a roadmap as to where we are heading (provided we include the document on the website).

OTOH, we already have GitHub issues to track feature requests and can use labels to organize them according to priority, etc., but it's not as user-friendly when it comes to garnering future roadmap.

**Comment by rempsyc** [comment]
Of course, I can't disagree with this brilliant idea hehehe 😝. I can imagine however that it could be trickier with managing 11 packages, since each one would probably have its own file (for manageability), and so it would be more maintenance work keeping all of them up to date. The organization and clarity benefits are really satisfying though. And yes, for newbies and users that read the pkgdown site (if we include it there), it's a gift. And it could (possibly) prevent some people from opening new issues if they see we are already headed there.

**Comment by etiennebacher** [comment]
FYI it's possible to make a sort of roadmap in github issues using milestones:

![image](https://user-images.githubusercontent.com/52219252/206737807-2292e9bf-0be4-4631-aa98-8b897d763f15.png)

**Comment by etiennebacher** [comment]
> I don't know if GitHub milestones could be smoothly integrated on the pkgdown website in a nice and user-friendly way

Hmm I wonder if it's possible to use the github API (cf the dashboard) to extract issues with milestones and create a "roadmap" page on pkgdown. This is only useful if milestones are used of course

**Comment by mattansb** [comment]
I have been using github issues on all packages here in this manner already ^_^

Are we specifically talking about having it appear on a pkgdown page? We can add a "wishlist" tag to GH issues, and then link to this from pkgdown in the menu bar. E.g., [all open "low priority" tagged issues on effect size](https://github.com/easystats/effectsize/labels/low%20priority%20%F0%9F%98%B4)

**Comment by rempsyc** [comment]
Last thing. Using the github issue titles might not be as user-friendly as writing it ourselves for our target audience, because the issue titles are targetted at developers, so they are not always clear because they were not written with that in mind (a few example titles from this repo: "model dashboard not lao", "Predicted values tab error", "Please make a conda metapackage!").

However, if our github integrated roadmap can use specially selected issues (milestones tags), we can be more selective in which issues we include, for example, only issues with clear and self-sufficient titles. If we can do all this (github API integration like the dashboard), once set, it would have the advantage of being automatized and relatively low maintenance. We could keep the current workflow and just add the milestone tag to issues we want to see on that to-do list as we create the issue.

**Comment by rempsyc** [comment]
(Also, nothing is preventing us from using manual to-do lists now, and as we take the time to figure out the GitHub milestone dashboard integration eventually)

**Comment by IndrajeetPatil** [comment]
TBH, this is not at all what I had in mind from a user-facing document like this. 

As a user who is not too deep in the weeds about easystats, GitHub, or software development, what I would like to know from a TODO document is what's the future roadmap for the package, without getting into as much details as a GitHub issue.

For example, what remains to be done before there is a stable (`1.0`) release.
